src/components/pages/StudentHome/StudentAchievements.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import Banner from '../../common/Banner';
+import StudentSidebar from './StudentSidebar';
+import '../../../styles/ParentStyles/index.less';
+import { Layout, Card } from 'antd';
+import 'antd/dist/antd.css';
+import brightIdea from '../../../img/Assets/Bright idea.svg';
+import caringHelper from '../../../img/Assets/Caring helper.svg';
+import coolCoder from '../../../img/Assets/Cool Coder.svg';
+import dailyChampion from '../../../img/Assets/Daily Champion.svg';
+import designThinker from '../../../img/Assets/Design Thinker.svg';
+import ecoConscious from '../../../img/Assets/Eco- conscious.svg';
+import organizedTaskmaker from '../../../img/Assets/Organized Taskmaker.svg';
+import teamworkCollaborator from '../../../img/Assets/Teamwork collaborator.svg';
+
+const StudentAchievements = () => {
+  const { Content } = Layout;
+  return (
+    <Layout>
+      <StudentSidebar active="dashboard" />
+      <Content className="achievements-container">
+        <Banner active="achievements" />
+        <a href="">
+          <Card className="achievement-card">
```

Comment:
You should create a separate component for achievements so that it can be hooked up to the backend.